jeffjxu/Wesley-Central-Access-master
# edit client when editing inquiry

able to edit client when editing an inquiry, also removed "Edit Client" button in inquiry view screen
How to test:
- go to New Inquiries
- Click edit on one of the inquiries
- edit the client information
- click save inquiry
- check if client information is correctly updated
- check if the edit client button is gone in the view screen
